.changeset/sweet-tips-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+'@api3/airnode-node': patch
+---
+
+remove MAXIMUM_SPONSOR_WALLET_REQUESTS
```

Comment:
nit: Capitalize remove

Reply:
Oh, 🤦 capitalized !== uppercase... I misunderstood what you mean.